charts/jenkins/README.md

```diff
@@ -144,20 +144,30 @@ Error: Deployment.apps "jenkins" is invalid: spec.selector: Invalid value: v1.La
 
 In order to upgrade, [uninstall](#uninstall-chart) the Jenkins Deployment before upgrading:
 
+## Change Log
+Visit the chart's [CHANGELOG](./CHANGELOG.md) to view the chart's release history. 
```

Comment:
it was suggested that a reference to the `CHANGELOG` be added
https://github.com/jenkinsci/helm-charts/pull/1#discussion_r479694259